packages/hooks/__tests__/use-throttle-render.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { defineComponent, nextTick, ref } from 'vue'
+import { mount } from '@vue/test-utils'
+import { afterEach, beforeEach, describe, expect, it } from 'vitest'
+import { useThrottleRender } from '../use-throttle-render'
+import type { VueWrapper } from '@vue/test-utils'
+
+const Comp = defineComponent({
+  setup() {
+    const loading = ref(false)
+    const throttled = useThrottleRender(loading, 1000)
+    // Test the settimeout branch clearly: trigger the watch to record the settimeout first, and then record the settimeout again when mount.
+    loading.value = true
+
+    return () => <div class="test-dom">{throttled.value.toString()}</div>
+  },
+})
+
+describe('useThrottleRender', () => {
```

Comment:
It seems like multiple test cases don't affect each other, perhaps using [describe.concurrent](https://vitest.dev/api/#describe-concurrent) to reduce running time.